obsRecorder.js

```diff
@@ -30,7 +31,7 @@ function initialize(win) {
 
 function initOBS() {
   console.debug('Initializing OBS...');
-  osn.NodeObs.IPC.host('obs-studio-node-example'); // Usually some UUIDs go there
+  osn.NodeObs.IPC.host(`obs-studio-node-example-${uuid()}`);
```

Comment:
Are unique identifiers required for MacOS or you added it along the way? Is there specific reason to use it now?

Reply:
Without the uuid the app only works on the first start, and freezes after every subsequent start. Not sure what the root cause is